test/performance/scale_from_zero_test.go

```diff
@@ -45,19 +45,22 @@ type stats struct {
 	avg time.Duration
 }
 
-func runScaleFromZero(t *testing.T, clients *test.Clients, ro *test.ResourceObjects) (time.Duration, error) {
+func runScaleFromZero(idx int, t *testing.T, clients *test.Clients, ro *test.ResourceObjects) (time.Duration, error) {
+	t.Helper()
 	deploymentName := names.Deployment(ro.Revision)
 
 	domain := ro.Route.Status.Domain
-	t.Log("Waiting for deployment to scale to zero.")
+	t.Logf("%d: waiting for deployment to scale to zero.", idx)
 	if err := pkgTest.WaitForDeploymentState(
 		clients.KubeClient,
 		deploymentName,
 		test.DeploymentScaledToZeroFunc,
 		"DeploymentScaledToZero",
 		test.ServingNamespace,
 		2*time.Minute); err != nil {
-		return 0, fmt.Errorf("Failed waiting for deployment to scale to zero: %v", err)
+		m := fmt.Sprintf("%d: failed waiting for deployment to scale to zero: %v", idx, err)
+		t.Log(m)
+		return 0, fmt.Errorf(m)
```

Comment:
nit: use `fmt.Errorf` above and `t.Log(err.Error())`?  This isn't a formatting call.

Reply:
Done, but in a slightly different vein.